docs/core/tools/dotnet-test.md

```diff
@@ -80,10 +80,18 @@ Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework.
 
 ## Options
 
-- **`-a|--test-adapter-path <ADAPTER_PATH>`**
+> [!WARNING]
+> Breaking changes in options:
```

Comment:
should we add "starting from?"

Reply:
Yes! I was thinking about it and missed it.